src/payment_tokens.json

```diff
@@ -0,0 +1,23 @@
+{
+  "base": { "0x833589fcd6edb6e08f4c7c32d4f71b54bda02913": "USDC" },
+  "sepolia": {
+    "0x1c7d4b196cb0c7b01d743fbc6116a902379c7238": "USDC",
+    "0xaa8e23fb1079ea71e0a56f48a2aa51851d8433d0": "USDT"
+  },
+  "mainnet": {
+    "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48": "USDC",
+    "0xdac17f958d2ee523a2206206994597c13d831ec7": "USDT"
+  },
+  "optimism": {
+    "0x0b2c639c533813f4aa9d7837caf62653d097ff85": "USDC",
+    "0x94b008aa00579c1307b0ef2c499ad98a8ce58e58": "USDT"
+  },
+  "arbitrum": {
+    "0xaf88d065e77c8cc2239327c5edb3a432268e5831": "USDC",
+    "0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9": "USDT"
+  },
+  "polygon": {
+    "0x3c499c542cef5e3811e1192ce70d8cc03d5c3359": "USDC",
+    "0xc2132d05d31c914a87c6611c10748aeb04b58e8f": "USDT"
+  }
```

Comment:
This is not needed, we can remove it
```suggestion
```

Reply:
I think your commit suggestion is wrong? Removing the last curly brackets errors on my computer.

Reply:
Yes need to remove some comma

Reply:
aaaaah I understand what you meant lol. You were referring to the arbitrum / polygon / optimism. The diff didn't make that clear lol. Removed in e87f27145bec381d8102012577852950762cf12e